pkg/inference/scheduling/loader.go

```diff
@@ -224,17 +232,17 @@ func (l *loader) evict(idleOnly bool) int {
 // It returns the number of remaining runners.
 func (l *loader) evictRunner(backend, model string, mode inference.BackendMode) int {
 	allBackends := backend == ""
-	for r, slot := range l.runners {
-		unused := l.references[slot] == 0
-		if unused && (allBackends || r.backend == backend) && r.model == model && r.mode == mode {
+	for r, runnerInfo := range l.runners {
+		unused := l.references[runnerInfo.slot] == 0
+		if unused && (allBackends || r.backend == backend) && r.modelID == model && r.mode == mode {
 			l.log.Infof("Evicting %s backend runner with model %s in %s mode",
```

Comment:
It would be useful if the logging statements in `loader.go` and `scheduler.go` retained the model ref as well (maybe in a standardized format like `ai/<modelRef>@sha256:<modelID>`).  It would make debugging significantly easier.

Reply:
@xenoscopic update [here](https://github.com/docker/model-runner/pull/98/commits/a23dc5e5142d03170b665748e13d2f41eaf82f86)